plugins/scaffolder-actions/scaffolder-backend-module-utils/src/actions/jsonata/json.ts

```diff
@@ -64,26 +71,29 @@ export function createJsonJSONataTransformAction() {
         properties: {
           result: {
             title: 'Output result from JSONata',
-            type: 'object',
+            type: 'object | string',
           },
         },
       },
     },
     async handler(ctx) {
+      let post: (rz: any) => any;
```

Comment:
maybe rename this to be a little bit more descriptive

Reply:
Later I saw how the file formats on `fs:parse` are handled with a `Record` but I don't know that that would be particularly helpful in this scenario. I will expand on the variable name.